File: Prácticas/P5/JAVA/Civitas/src/civitas/OperacionInmobiliaria.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

/**
 *
 * @author Jose Padial
 */
public class OperacionInmobiliaria {
    private int indicePropiedad;
    private OperacionesInmobiliarias operacion;
    
    public OperacionInmobiliaria(int indice, OperacionesInmobiliarias operac){
        indicePropiedad = indice;
        operacion = operac;
    }
    
    public int getIndicePropiedad(){
        return indicePropiedad;
    }
    
    public OperacionesInmobiliarias getOperacion(){
        return operacion;
    }
    
}

File: Prácticas/P4/RUBY/Civitas/lib/operacion_inmobiliaria.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative"operaciones_inmobiliarias.rb"

module Civitas
class Operacion_inmobiliaria
  
  def initialize(index,operac)
    @indice_propiedad = index
    @operacion = operac
  end
  
  def get_indice_propiedad
    return @indice_propiedad
  end
    
  def get_operacion
    return @operacion
  end
    
end
end
